packages/cli/src/commands/members/index.ts

```diff
@@ -0,0 +1,88 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {RoleCompletion} from '@heroku-cli/command/lib/completions'
+import {ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+
+const _ = require('lodash')
+
+type MemberWithStatus = Heroku.TeamMember & { status?: string }
```

Comment:
Based on the usage below, the `status` type seems to be `status: '' | 'pending'`. Are other values allowed here?

Reply:
No, we hardcode the status as "pending" on line 58. The status entry only gets added for the `teamInvites` object, so it's an optional entry for the members object and should only equal "pending" when it is included.